internal/compose/compose.go

```diff
@@ -275,54 +275,58 @@ func (p *Project) WaitForHealthy(opts CommandOptions) error {
 		return err
 	}
 
+	timeout := time.After(10 * time.Minute)
```

Comment:
Please move this timeout to const at the top of the file and name it accordingly.